parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -490,6 +616,7 @@ mod benches {
 	define_benchmarks!(
 		[frame_system, SystemBench::<Runtime>]
 		[pallet_balances, Balances]
+		[pallet_session, SessionBench::<Runtime>]
```

Comment:
Why are we duplicating this benchmark - is this a mistake?

Reply:
good catch, thank you, will be removed, probably bad merge...